modules/command/common/send-telemetry-command.js

```diff
@@ -32,12 +21,18 @@ class SendTelemetryCommand extends Command {
         }
 
         this.telemetryHubModuleManager.aggregateTelemetryData()
-            .then(jsonld =>
-            {
-                if (jsonld)
-                    this.publishService.publish(JSON.stringify(jsonld), '.json', [], [`ot-telemetry-${Math.floor(new Date() / (60 * 60 * 1000))}`], true, null)
+            .then((jsonld) => {
+                if (jsonld) {
+                    Models.handler_ids.create({
+                        status: 'PENDING',
+                    }).then((insertedObject) => {
+                        this.publishService.publish(JSON.stringify(jsonld), '.json', [], [`ot-telemetry-${Math.floor(new Date() / (60 * 60 * 1000))}`], true, insertedObject.dataValues.handler_id);
```

Comment:
We still haven't discussed about that but it should be less than an hour. Let's leave it for now